crates/oxc_regular_expression/src/ast.rs

```diff
@@ -238,6 +238,7 @@ pub struct CharacterClass<'a> {
     pub span: Span,
     pub negative: bool,
     pub kind: CharacterClassContentsKind,
+    pub strings: bool,
```

Comment:
We probably need some documentation around these fields, I don't know what they mean 😢

Reply:
OK, I'll review the AST documentation in another PR. 🏃🏻